client/starwhale/core/instance/view.py

```diff
@@ -59,7 +59,7 @@ def logout(self, instance: str) -> None:
             sys.exit(1)
 
     def info(self, uri: str = "") -> t.Dict:
-        instance = Instance(uri or self.current_instance)
+        instance = Instance(uri)
```

Comment:
Will this change lead to breaking change, because the Instance constructor don't deal with the uri=""?